Wallet Storage Rework

gy1ta23 opened the pull request:
🆑
tweak: Wallets can hold less things they shouldn't be able to, and can hold more things they should.
/🆑

### I'm back, bitches.

And upon my awakening, as the lord (me) foretold, I bring change, glorious, glorious change. The ethereal storage compartment so long ago added to the wallet has been removed, and in exchange, things that should fit in wallets shall now actually fit in wallets. Your prayers, long gone unheard, have been answered, at last. Rejoice in the fields and in the streets, in the cities and towns. I shall allow it, **this once.**

SierraKomodo reviewed (commented):
What's the reasoning for these changes? Some of this feels a bit arbitrary. For example, removing the pAI card but then allowing integrated circuits, stock parts, and encryption keys?

gy1ta23 commented:
The pAI cards, and other removed items, are too large or highly impractical to fit in a wallet. The other objects, by contrast, are not.

Spookerton commented on code/game/objects/items/weapons/storage/wallets.dm:
```diff
@@ -10,38 +10,35 @@
 		/obj/item/clothing/accessory/armor/tag,
 		/obj/item/clothing/ring,
-		/obj/item/passport
+		/obj/item/integrated_circuit,
```
nop

Spookerton commented on code/game/objects/items/weapons/storage/wallets.dm:
```diff
@@ -10,38 +10,35 @@
-		/obj/item/passport
+		/obj/item/integrated_circuit,
+		/obj/item/passport,
+		/obj/item/stock_parts,
+	)
+	cant_hold = list(
+	    /obj/item/stock_parts/matter_bin,
+	    /obj/item/stock_parts/circuitboard,
```
nop

Spookerton commented on code/game/objects/items/weapons/storage/wallets.dm:
```diff
@@ -10,38 +10,35 @@
 		/obj/item/device/flashlight/slime,
 		/obj/item/seeds,
 		/obj/item/material/coin,
-		/obj/item/dice,
```
nop

Spookerton commented on code/game/objects/items/weapons/storage/wallets.dm:
```diff
@@ -10,38 +10,35 @@
 		/obj/item/disk,
 		/obj/item/implant,
-		/obj/item/implanter,
-		/obj/item/flame,
```
nop

Spookerton commented:
Rest seems `fine`. I guess.

Commit:
wallet storage rework


Requested changes


Requested changes 2 


Requested changes 3


Requested changes


Requested changes


Requested changes (again)


Removing ecig carts (because I'm fucking blind)

gy1ta23 commented:
The requested changes have been enacted (I'm 95% sure they have been, at least).
Love you too spook
